udms/e2e/firestore/import_db_cloud.js

```diff
@@ -0,0 +1,99 @@
+// Point json_data_source to the json file with data to be populated into the DB
+// Point db_collection to the collection name/topmost field of json object being
+// populated.
+// If script fails with permission issues, run:
+// 'gcloud auth application-default login'
+
```

Comment:
You can import functions from the original script. It looks like the 2 are almost identical.

Reply:
Changed.